docs/source/workflow/build-run-queries.mdx

```diff
@@ -2,134 +2,150 @@
 title: Build and run queries against Apollo Server
 ---
 
-In non-production environments, Apollo Server 3 serves the following landing page from its base URL (`http://localhost:4000` by default):
+In non-production environments, Apollo Server 4's landing page is an embedded version of [Apollo Sandbox](/studio/explorer/sandbox) (served at `http://localhost:4000` by default):
 
-<img class="screenshot" src="../images/as-landing-page.jpg" alt="Apollo Server default landing page" width="350"/>
+<img class="screenshot" src="../images/sandbox.jpeg" alt="Apollo Sandbox" />
```

Comment:
kinda curious why JPG and not PNG for screenshot

Reply:
Oh, that's the image we already were using. I can convert it to another file format(or take a new screenshot) if you feel like we should align on having screenshots use PNGs?

Reply:
Well no point converting a pre-existing file from JPG. PNG is a precise recording of every pixel in the screenshot, whereas JPG is a lossy format designed for representing the complexity of *photographs* in an efficient manner. But eh, I zoomed in and it doesn't look particularly affected by artifacts. I'd suggest PNG for new screen shots though.

Reply:
Sounds good! I'll do that from now on!